tgui/packages/tgui/interfaces/RoboticsControlConsole.js

```diff
@@ -70,16 +70,22 @@ const Cyborgs = (props, context) => {
             )}
             <Button.Confirm
               icon={cyborg.locked_down ? 'unlock' : 'lock'}
+              disabled={!cyborg.can_lock&&!cyborg.locked_down}
               color={cyborg.locked_down ? 'good' : 'default'}
               content={cyborg.locked_down ? "Release" : "Lockdown"}
               onClick={() => act('stopbot', {
                 ref: cyborg.ref,
               })} />
             <Button.Confirm
               icon="bomb"
-              content="Detonate"
-              color="bad"
-              onClick={() => act('killbot', {
+              disabled={!cyborg.locked_down&&!cyborg.can_blow}
+              content={!cyborg.can_blow&&cyborg.locked_down
+                ?"Arming up"
+                :"Detonate"}
+              color={cyborg.can_blow ? "bad"
+                :cyborg.locked_down ? "yellow" : "grey"}
+              onClick={() => act(cyborg.can_blow
+                ?'killbot' : "", {
```

Comment:
Must be properly formatted, use `bin/tgui --fix`

Reply:
I used the command, it changed nothing. Could you elaborate on what i need to fix in that case?